docs/source/cluster_management/sweep/background-sweep.rst

```diff
@@ -0,0 +1,66 @@
+.. _background_sweep:
+
+Background Sweep
+================
+
+.. warning::
+
+   Background sweep is currently considered to be an experimental feature, and is disabled by default.
+   If you are interested in trialling background sweep, please contact the AtlasDB team.
+
+
+How Background Sweep Works
+--------------------------
+
+The Background Sweep Job works by sweeping one table at a time.
+The Background Sweep Job determines which table to sweep by estimating which would be most beneficial based on I/O activity and frequency, considering the following criteria:
+
+- The number of cells written to a table since it was last swept.
+- The number of cells that were deleted the last time a table was swept.
+- The number of cells that were not deleted the last time a table was swept.
+- The amount of time that has passed since the it was last swept.
+
+
+Additional logging for Background Sweep
+---------------------------------------
+
+By default, the background sweeper only logs errors. If you'd like to watch the background sweeper's progress, add the following in ``atlasdb.log.properties``:
+
+  ::
+
+    #--------------------------------------------------------------------------------
+    # Sweep Logging
+    #--------------------------------------------------------------------------------
```

Comment:
I'm not an expert on log4j, but I'm pretty sure at least line 35 is wrong -- `debug` is more detailed than `info`. (Also FWIW I would recommend suggesting `info` as a default on line 36).